pajbot/modules/wide_emote_limit.py

```diff
@@ -0,0 +1,121 @@
+import logging
+
+from pajbot.managers.handler import HandlerManager
+from pajbot.modules import BaseModule, ModuleSetting
+
+log = logging.getLogger(__name__)
+
+
+class WideEmoteLimitModule(BaseModule):
+    ID = __name__.split(".")[-1]
+    NAME = "Wide Emote Limit"
+    DESCRIPTION = "Times out users who post too many wide 7TV or FFZ emotes"
```

Comment:
Add a `LONG_DESCRIPTION` or whatever the field is called explaining that we only know the width of 7tv and ffz emotes, bttv and twitch emotes don't give us their size